### ISSUE TITLE
Välivarastojen siivoaminen RDA konversioiden jälkeen

### ISSUE THREAD

**Issue by johannaraisa** [KohaSuomi/Koha#1508]
### Tukipyyntö

Välivarastot vievät palvelintilaa ja ne pitäisi siivota ainakin ennen YSO-konversioita.

**Comment by johannaraisa** [comment]
Siivouksen seuranta:

- [ ] OUTI
- [ ] Vaara
- [ ] Vaski
- [ ] Lumme
- [ ] Lappi
- [ ] Siili
- [ ] Helle
- [ ] Kyyti
- [ ] Kirkes
- [ ] Lastu

**Comment by johannaraisa** [comment]
Laitoin Tätiin päälle siivousskriptin, kun tuo on aika hidasta käsin kannasta. Siellä siivotaan illalla kaikki 60 päivää vanhemmat välivarastotietueet. Tuskin enää sitä vanhempia tarvitsee katsella. Katson huomenna miten meni ja siivoan sitten muualtakin tuolla noita tietueita pois.

**Comment by johannaraisa** [comment]
Muuten ihan hyvä, mutta BTJ säilyttää palvelimellaan ikuisuuden noita päiväpaketteja joten ne on tulleet uudestaan välivarastoon ja tietenkin niitä on yritetty valuttaa tänä aamuna. Keskeytin sen ja yritin saada käsittelemään vain ne viimeisimmät.

**Comment by AnttHeik** [comment]
Jeps, meille OUTIin tuli ainakin 14 välilehteä tietueita uudestaan. Onneks suurin osa vain ehkä peruspäivityksiä. Tämän aamun varsinaiset tietuepaketit ovat jääneet kuitenkin valumatta.

**Comment by johannaraisa** [comment]
Joo, katselin että nyt kun ne yhdistyy 001 ja 003 kentillä, niin päivittynyt se BTJ:n versio, muutama tupla jos oli Melinda j olemassa. Se on aamulla jauhanut niitä päivityksiä ja ei ole päässyt vielä loppuun. Yritän saada ne viimeisimmät tulemaan.

**Comment by johannaraisa** [comment]
Hyvin ainakin nuo vanhat välivarastot poistui, olisi vaan muistanut tuon yksityiskohdan Tätin osalta. Jatkan siivoustyötä ensi viikolla.

**Comment by AnttHeik** [comment]
Hei, tänä aamuna oli OUTIin taas paljon tietueita valunut uudestaan. Liittyykö mahdollisesti tähän vai onkohan tuo jokin muu homma?

**Comment by johannaraisa** [comment]
Taitaa liittyä, yksi ylimääräinen paketti näköjään uudestaan. Yritin kyllä säätää ettei hakisi niin kauas noita, pitää varmaan tiukentaa rajaa vielä.

**Comment by johannaraisa** [comment]
Kirkekselle ja Lastulle ei ole vielä edes tehty RDA-konversiota. Ehkä niistä voisi omat tiketit, joten suljetaan tämä.
